CHANGELOG.md

```diff
@@ -91,6 +91,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Fixed
 - Fix flaky ResourceAwareTasksTests.testBasicTaskResourceTracking test ([#8993](https://github.com/opensearch-project/OpenSearch/pull/8993))
 - Fix memory leak when using Zstd Dictionary ([#9403](https://github.com/opensearch-project/OpenSearch/pull/9403))
+- Add support to clear archived index setting ([#9019](https://github.com/opensearch-project/OpenSearch/pull/9019))
```

Comment:
@ankitkala This entry was put in the `Unreleased 3.0` section on main instead of the 2.x section: https://github.com/opensearch-project/OpenSearch/blob/762c70bf3b0dd5cbaf2cac04d52dd8abded08704/CHANGELOG.md?plain=1#L75

which is likely the cause of the backport conflict. Can you fix up that entry on main?